docs/CHANGELOG.rst

```diff
@@ -24,13 +19,16 @@ Changed
 * Update differential expression (DE) processors
 * Update ``generate_diffexpr_cuffdiff`` django-admin command
 * Save gene_id source to ``output.source`` for DE, expression and related objects
+* Update sample descriptor schema
 
 Added
 -----
 * Process definition documentation style and text improvements
+* Add tests to ensure generators produce the same results
```

Comment:
This is unrelated to the descriptor schemas. This change in to fix an error in the change-log that is present in the current master. There were two "Added" sections introduced by previous merge.